src/pages/challenges/2018-10-26a.md

```diff
@@ -8,7 +8,7 @@ tags:
   - accordion
   - pure
   - CSS
-difficulty: Beginner
+difficulty: 3
```

Comment:
I see what you are doing here, though will that make searching by difficulty harder (e.g. searching for "3" instead of "Beginner")?